src/base/UMain.pas

```diff
@@ -262,6 +265,14 @@ procedure Main;
       Display.CurrentScreen^.FadeTo( @ScreenOptionsRecord );
     end;
 
+    //------------------------------
+    // Start Webserver
+    //------------------------------
+    Log.LogStatus('Webserver', 'Initialization');
+    // Create and start the web server
+    Server := TWebServer.Create(8091);
```

Comment:
please make the port configurable, but set it to strictly only listen on localhost. If people want to have it reachable over network, this forces them to put at the very least a tiny bit of effort into setting up a reverse proxy, which then hopefully has much less attack surface than the old UWebServer.